libraries/AP_IOMCU/AP_IOMCU.cpp

```diff
@@ -100,6 +104,34 @@ void AP_IOMCU::init(void)
     initialised = true;
 }
 
+/*
+  disable the IOMCU operation by erasing iofirmware
```

Comment:
I think this should have a different name given the fairly radical function of its operation

Reply:
thoughts on name, erase is already taken.

Reply:
changing it to wipe

Reply:
removed completely